elastic/conf.yaml.example

```diff
@@ -34,6 +34,7 @@ instances:
     # password: password
     # cluster_stats: false
     # pshard_stats: false
+    # pshard_timeout_graceful: false  # continue gracefully if pshard stats TO
```

Comment:
this should read `pshard_graceful_timeout` to be consistent with what's in the check :)